packages/react-dom/src/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -954,50 +954,38 @@ describe('ReactDOMComponent', () => {
     });
 
     it('should warn against children for void elements', () => {
-      var container = document.createElement('div');
-
-      expect(function() {
+      const container = document.createElement('div');
+      let caughtErr;
+      try {
         ReactDOM.render(<input>children</input>, container);
-      }).toThrowError(
+      } catch (err) {
+        caughtErr = err;
+      }
+      expect(caughtErr).not.toBe(undefined);
+      expect(normalizeCodeLocInfo(caughtErr.message)).toContain(
         'input is a void element tag and must neither have `children` nor ' +
           'use `dangerouslySetInnerHTML`.',
+        '\n    in input (at **)',
       );
     });
 
     it('should warn against dangerouslySetInnerHTML for void elements', () => {
-      var container = document.createElement('div');
-
-      expect(function() {
+      const container = document.createElement('div');
+      let caughtErr;
+      try {
         ReactDOM.render(
           <input dangerouslySetInnerHTML={{__html: 'content'}} />,
           container,
         );
-      }).toThrowError(
-        'input is a void element tag and must neither have `children` nor use ' +
-          '`dangerouslySetInnerHTML`.',
-      );
-    });
-
-    it('should include owner rather than parent in warnings', () => {
```

Comment:
Removed this test since it's not really useful in component stacks (where we have both).
Would be nice to mark owner in some special way in stacks but that's a separate issue.